rollup.config.ts

```diff
@@ -0,0 +1,18 @@
+import typescript from "rollup-plugin-typescript2";
+
+export default {
+  input: "src/index.ts",
+  output: [
+    {
+      file: "dist/index.js",
+      format: "cjs",
+      sourcemap: true
```

Comment:
Are sourcemaps still useful for node only libraries?

Reply:
@kulak-at might better answer this as he's more familiar with Node development.

Reply:
Spoke to Kacper, might as well leave them in 👍